ui/components/ui/icon/network-tree-icon.component.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function NetWorkTreeIcon({
```

Comment:
- network should be one word
- maybe rename as `NetworkTreeIcon`

Reply:
fixed in 17835259df6c1e0ae956afb2da43e714ed902767